data/create_users

```diff
@@ -5,5 +5,5 @@ encrypted_password="$2"
 
 for i in `seq 1 $n_users` ; do
 	num=`printf "%02d" $i`
```

Comment:
I though I fixed it by using tab, but forgot that tab in my vim is set to do 4 spaces